requirements.txt

```diff
@@ -20,6 +20,7 @@ pillow==5.2.0
 boto3==1.7.80
 botocore==1.10.80
 django-storages==1.6.6
-python-dateutil
```

Comment:
We use dateutil at 2 places:
1. views.py -- line 31
2. sms.py -- line 18
Please explain why do you want to replace it with python-dateutil